ocaml/libs/vhd/vhd_format/f.ml

```diff
@@ -18,6 +18,10 @@ let sector_size = 512
 
 let sector_shift = 9
 
+(** This is defined to be the same as sync_limit in channels.ml due to circular 
+    dependencies*)
+let sync_limit = Int64.(mul 4L (mul 1024L 1024L))
```

Comment:
It's worth documenting this is in bytes

Reply:
Or let code do the talking:
```
let mib n = 
  let ( ** ) = Int64.mul in
  Int64.(1024L ** 1024L ** of_int n)

let sync_limit = 4 mib
```

Reply:
you mean `let sync_limit = mib 4` right?

Reply:
Indeed